src/pybind/mgr/rbd_support/mirror_snapshot_schedule.py

```diff
@@ -48,9 +48,11 @@ def __del__(self) -> None:
         self.wait_for_pending()
 
     def wait_for_pending(self) -> None:
-        self.log.debug("CreateSnapshotRequests.wait_for_pending")
         with self.lock:
             while self.pending:
+                self.log.debug(
+                    "CreateSnapshotRequests.wait_for_pending: "
+                    "{} snapshot requests pending".format(len(self.pending)))
```

Comment:
Strictly speaking, what is pending are images, with exactly one snapshot each.  How about the following to make it a bit shorter and also be explicit about a case when nothing is pending:
```
    def wait_for_pending(self) -> None:
        with self.lock:
            while self.pending:
                self.log.debug(
                    "CreateSnapshotRequests.wait_for_pending: {} images".format(
                        len(self.pending)))
                self.condition.wait()
        self.log.debug("CreateSnapshotRequests.wait_for_pending: done")
```

Reply:
Got it. thanks!